internal/pkg/manifest/app_manifest.go

```diff
@@ -35,15 +35,11 @@ type AppImage struct {
 	Build string `yaml:"build"` // Path to the Dockerfile.
 }
 
-// CreateApp returns a manifest object based on the application's type.
-// If the application type is invalid, then returns an ErrInvalidManifestType.
-func CreateApp(appName, appType, dockerfile string) (archer.Manifest, error) {
-	switch appType {
-	case LoadBalancedWebApplication:
-		return NewLoadBalancedFargateManifest(appName, dockerfile), nil
-	default:
-		return nil, &ErrInvalidAppManifestType{Type: appType}
-	}
+// CreateAppManifestProps contains all properties for creating a new manifest.
+type CreateAppManifestProps struct {
```

Comment:
nit: can we move this to `lb_fargate_manifest.go`? and maybe rename to `LBFargateManifestProps`